tradeit-android-sdk/src/androidTest/kotlin/it/trade/android/sdk/ApplicationTest.kt

```diff
@@ -0,0 +1,9 @@
+package it.trade.android.sdk
+
+import android.app.Application
+import android.test.ApplicationTestCase
+
+/**
+ * [Testing Fundamentals](http://d.android.com/tools/testing/testing_android.html)
+ */
+class ApplicationTest : ApplicationTestCase<Application>(Application::class.java)
```

Comment:
this `ApplicationTest` is not used, should be removed too.

Reply:
Removed